collections/made-in-brazil/index.md

```diff
@@ -13,6 +13,9 @@ items:
   - RocketChat/Rocket.Chat
   - tsuru/tsuru
   - goreleaser/goreleaser
+  - Universidade-Livre/ciencia-da-computacao
+  - cuducos/minha-receita
+  - aprenda-frontend
```

Comment:
It doesn't look like this org exists. Is there a typo?

Reply:
(Also, hello! 👋🏻 )

Reply:
hmmmm, must be, also hiya!!!! taking alook

Reply:
yeap it was my bad! fixed and added one more felipeorlando/aprenda-rubyonrails